### PR TITLE
[#192] Add move SFX

### PR DESCRIPTION
**Pull request by benshumaker**
Closes: #192

### CODE REVIEW

File: frontend/packages/web/src/react-app-env.d.ts [hunk 1]
```diff
@@ -1 +1,2 @@
 /// <reference types="react-scripts" />
+declare module '*.mp3';
```
Comment (izakfr) [inline frontend/packages/web/src/react-app-env.d.ts:2]: What does this do?

File: frontend/packages/web/src/components/features/Subscription/Subscription.tsx [hunk 2]
```diff
@@ -65,6 +65,9 @@ export default function Subscription() {
 
   const handleGameMessage = (message: any) => {
     setGame({ ...game, ...(message.data as Game) });
+    if (message.data?.gameboard_state) {
```
Comment (izakfr) [inline frontend/packages/web/src/components/features/Subscription/Subscription.tsx:68]: So this plays every time we receive an update that includes gameboard state?

**Review by izakfr — CHANGES_REQUESTED** [review]
Had some questions about when the sound plays.

File: frontend/packages/web/src/react-app-env.d.ts (hunk 1 above)
Comment (benshumaker) [reply frontend/packages/web/src/react-app-env.d.ts:2]: This tells typescript that we want to be able to import .mp3 files. I don't know how it works beyond that. Without this statement, the typescript pukes when you try to import an mp3 file.

File: frontend/packages/web/src/components/features/Subscription/Subscription.tsx (hunk 2 above)
Comment (benshumaker) [reply frontend/packages/web/src/components/features/Subscription/Subscription.tsx:68]: Yes. Originally, I had this running when they made a move. But then the game wouldn't make a sound when the opponent made a move. I switched to this version and removed the original sound in square.tsx. We have to put something in the websocket-game-state processing to make a sound on opponent moves.

Comment (benshumaker) [reply frontend/packages/web/src/components/features/Subscription/Subscription.tsx:68]: Does this not work as a timing? Are there updates that aren't moves that have new game states? Or should we be future proofing against non-delta updates? It's not much work to rework this, but I wasn't sure what a better timing/condition would be.

Comment (izakfr) [reply frontend/packages/web/src/components/features/Subscription/Subscription.tsx:68]: Yeah this seems fine, I don't want to create a more complicated solution right now.

**Review by izakfr — APPROVED** [review]

**Pull request merged**